### PR TITLE
fix(edit-content): Binary Field working in old Editor

### LINKED ISSUE #29127: Binary field visibility issue when creating new content
### Problem Statement

Users are currently unable to see the binary field when creating new content. This issue prevents users from uploading or linking file assets as part of their content creation process, impacting their ability to manage and utilize file assets effectively.

image

### Steps to Reproduce

- Log into the content management system with the necessary credentials.
- Navigate to the section for creating new content.
- Attempt to locate and interact with the file asset field.
- Observe that the file asset field is not visible or accessible.

### Acceptance Criteria

- **Visible File Asset Field**: Ensure that the file asset field is visible and accessible when users are creating new content.
- **Functionality:** The file asset field should function correctly, allowing users to upload or link file assets as needed.

### dotCMS Version

Tested on QA-Master // Docker // FF 

### Proposed Objective

Quality Assurance

### Proposed Priority

Priority 1 - Show Stopper

### PR DESCRIPTION
**Pull request by nicobytes**
### Parent issue

https://github.com/dotCMS/core/issues/29127

### Proposed Changes
* Remove the `ControlContainer` injector because as a web component this one doesn't have that context.
* Use `@Input` instead of signal inputs, because this type of input doesn't work well with web components yet.

### Checklist
- [x] Tests
- [x] Translations
- [x] Security Implications Contemplated (add notes if applicable)

### Additional Info

Here is a video showing the binary field working in the old editor and the new editor

https://github.com/dotCMS/core/assets/7611944/287ca76e-5b6d-4e6c-a052-1563d56dc0be

This PR fixes: #29127

### CODE REVIEW

File: core-web/apps/dotcms-binary-field-builder/src/app/app.module.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ const CONTENTTYPE_FIELDS: ContenttypeFieldElement[] = [
         MonacoEditorModule
     ],
     providers: [DotMessageService, DotUploadService]
+    // bootstrap: [AppComponent]
```
Comment (hmoreras) [inline core-web/apps/dotcms-binary-field-builder/src/app/app.module.ts:36]: remove ?

Comment (oidacra) [inline core-web/apps/dotcms-binary-field-builder/src/app/app.module.ts:36]: delete.

**Review by oidacra — APPROVED** [review]

**Review by hmoreras — APPROVED** [review]

File: core-web/apps/dotcms-binary-field-builder/src/app/app.module.ts (hunk 1 above)
Comment (nicobytes) [reply core-web/apps/dotcms-binary-field-builder/src/app/app.module.ts]: Fixed!

Comment (nicobytes) [reply core-web/apps/dotcms-binary-field-builder/src/app/app.module.ts]: Fixed!

**Pull request merged**